api/sip/client.py

```diff
@@ -842,8 +847,10 @@ def parse_patron_status(cls, status_string):
 
     def now(self):
         """Return the current time, formatted as SIP expects it."""
+        tz_spaces = self.dialect_config.tz_spaces
         now = utc_now()
-        return datetime.datetime.strftime(now, "%Y%m%d0000%H%M%S")
+        zzzz = " " * 4 if tz_spaces else "0" * 4
+        return datetime.datetime.strftime(now, f"%Y%m%d{zzzz}%H%M%S")
```

Comment:
Can we add a small test for this that makes sure `now()` formats the date as we would expect based on the dialect selected?

Reply:
👍